pkg/ruler/ruler.go

```diff
@@ -80,6 +80,8 @@ type Config struct {
 	RulePath string `yaml:"rule_path"`
 
 	// URL of the Alertmanager to send notifications to.
+	// If your are configuring the ruler to send to a Cortex Alertmanager,
```

Comment:
Nice addition. If you move this to the CLI flag description, than it would be part of the auto-generated config.

Reply:
Yea would be nice to have  this in auto generated doc. However, it's also nice to have this as inline doc, so I will get this merged and we can work on have it auto generated later on.